subcommands/waves/init.go

```diff
@@ -100,6 +113,15 @@ func doInitWave(cmd *cobra.Command, args []string) {
 		targets.Targets[name] = file
 	}
 
+	if len(prune) > 0 {
+		for _, v := range prune {
+			if v == version {
+				subcommands.DieNotNil(fmt.Errorf("Cannot prune current version"))
+			}
+		}
```

Comment:
Maybe, `if slices.Contains(prune, version)`?

Reply:
yes we are on high enough golang version.